fuuk/people/admin/forms.py

```diff
@@ -49,3 +50,12 @@ class Meta:
             'type', 'year', 'title', 'presenter',  # talk data
             'publication', 'volume', 'page_from', 'page_to', 'editors', 'place'  # abstract collection data
         )
+
+
+class GrantCollaboratorInlineFormSet(forms.BaseInlineFormSet):
+    def clean(self):
+        super(GrantCollaboratorInlineFormSet, self).clean()
+        investigator_human = self.instance.investigator_human
+        for form in self.forms:
+            if form.cleaned_data and investigator_human and investigator_human == form.cleaned_data['human']:
```

Comment:
* Check for `form.cleaned_data` seems useless.
 * `form.cleaned_data['human']` will raise `KeyError` if the human field is invalid.